cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/holder/BrokerLoad.java

```diff
@@ -89,7 +89,7 @@ public void recordMetric(CruiseControlMetric ccm) {
    * Check whether all dot handled topic metrics are available for the given dot-handled topic name.
    *
    * @param dotHandledTopic Dot-handled topic name.
-   * @return True if all dot handled topic metrics are available for the given dot-handled topic name, false otherwise.
+   * @return True if all dot handled topic metrics are available for the given dot-handled topic name, {@code false} otherwise.
```

Comment:
```suggestion
   * @return {@code true} if all dot handled topic metrics are available for the given dot-handled topic name, {@code false} otherwise.
```